src/components/list/examples/list-badge-icons.tsx

```diff
@@ -22,28 +22,28 @@ export class BadgeIconsListExample {
             secondaryText: '2-4 players',
             value: 2,
             icon: 'alien',
-            iconColor: 'var(--lime-green)',
+            iconColor: 'rgb(var(--color-lime-light))',
```

Comment:
You had missed the ones in this file… maybe you only searched for colours in `.scss` files?

Reply:
Hm.... I'm pretty sure I changed it :/ But probably missed it.

I never searched for any specific keyword. Went through everything line by line :)